### ISSUE TITLE
Yet another BMC virtual Ethernet device that needs to be ignored

### ISSUE THREAD

**Issue by bladernr** [canonical/checkbox#628]
### Bug Description

Ages ago it was decided that we should handle these on a case by case basis, so this is yet another BMC fake ethernet device that the network resource needs to learn to not label with NETWORK so that it is skipped during testing:

Category: NETWORK
Interface: enx8ed400a2881b
Product: Virtual Ethernet.
Vendor: American Megatrends, Inc.
Driver: cdc_ether
Driver Version: 
Path: /devices/pci0000:00/0000:00:14.0/usb1/1-11/1-11.3/1-11.3:2.0
Id: [046b:ffb0]
Mac: 8e:d4:00:a2:88:1b
Carrier Status: Connected
Ipv4: ***NOT CONFIGURED***
Ipv6: fe80::8cd4:ff:fea2:881b/64
Speed: -1
Supported Modes: 
	Not reported
Advertised Modes: 
	Not reported
Partner Modes: 

### To Reproduce

Run checkbox on the machine (or similar) to this one https://certification.canonical.com/hardware/202307-31883
BMC virtual ethernet is labled as NETWORK and checkbox attempts to test it

### Environment

See link above for details on evnrionment

### Relevant log output

```shell
See summary for relevant info
```
